src/DTF_ESP8266Update.cpp

```diff
@@ -68,6 +68,30 @@ void setSystemTime()
     #endif
 }
 
+void DTF_ESP8266Update::DTF_UpdateStart() {
+    #ifdef DTF_DEBUG
+    Serial.println("Callback: OTA Update Started");
+    #endif
+}
+
+void DTF_ESP8266Update::DTF_UpdateFinished() {
+    #ifdef DTF_DEBUG
+    Serial.println("Callback: OTA Update Finished Successfully");
+    #endif
+}
+
+void DTF_ESP8266Update::DTF_UpdateProgress(int _current, int _total) {
+    float percentage = ((float)_current/(float)_total) * 100;
+
+    #ifdef DTF_DEBUG
+    Serial.print("[");
+    for (int i=1; i<=(int)percentage; i++) {
+        Serial.print("+");
+    }
+    Serial.println("]");
+    #endif
```

Comment:
This is quite noisy on the serial output. This looks a lot cleaner and still gives progress information

```cpp
    static int last_percentage = 0;
    if (last_percentage >= 100)
    {
      return;
    }
    
    int percentage = (int)((float)_current/(float)_total * 100);
    
    if((percentage >= (last_percentage + 10)) || percentage == 100)
    {
      last_percentage = percentage;
      Serial.print(percentage);
      Serial.print("%..");
      if(percentage >= 100)
      {
        Serial.println();
      }
    }
```